ui/src/app/views/quick-tour-modal/component.css

```diff
@@ -9,7 +9,7 @@
   z-index: 1040;
 }
 
-.main {
+:host::ng-deep .main {
```

Comment:
We can do this as a follow up, but I propose that we move this to what I am proposing, where we use JS/TS objects, rather than CSS classes, which dodges the need for the :host::ng-deep.

Reply:
am totally on board, but let's do it as a follow up PR. i can make a ticket for it.